components/accordion.js

```diff
@@ -55,23 +57,26 @@ function AnimatedAccordionItemHeading({ expanded, children, ...rest }) {
     rotate: expanded ? 0 : -90,
     transformOrigin: "center",
   });
+
   return (
     <Box as={_AccordionItemHeading} {...rest}>
       <Box
         as={_AccordionItemButton}
         variant="accordion.heading"
         sx={{
-          backgroundImage({ colors: { accent, accentComplement } }) {
-            return `linear-gradient(90deg, ${accent} 0%, ${accentComplement} 100%, ${accentComplement} 100%)`;
-          },
+          backgroundImage: `linear-gradient(90.13deg, var(--theme-ui-colors-accent, ${accColor}) 49.89%, var(--theme-ui-colors-accentComplement, #517aea) 93.37%)`,
```

Comment:
Not sure I like how these colors look like. I'd probably stick to the orange color identity.